src/coremods/core_oper/core_oper.cpp

```diff
@@ -49,13 +37,28 @@ class CoreModOper : public Module
 	void ReadConfig(ConfigStatus& status) CXX11_OVERRIDE
 	{
 		ConfigTag* security = ServerInstance->Config->ConfValue("security");
-		cmdkill.hidenick = security->getString("hidekills");
-		cmdkill.hideuline = security->getBool("hideulinekills");
+		std::string hidenick = security->getString("hidekills");
+		bool hideuline = security->getBool("hideulinekills");
+
+		ConfigTag* tag = ServerInstance->Config->ConfValue("power");
+		// The hash method for *BOTH* the die and restart passwords
+		const std::string hash = tag->getString("hash");
+		const std::string diepass = tag->getString("diepass", ServerInstance->Config->ServerName);
```

Comment:
You should ensure these are at least one character log.

Reply:
Isn't an empty hash="" valid for plaintext passwords?

Reply:
I was referring to diepass and restartpass not hash.

Reply:
ah, yep added